src/strings/de.json

```diff
@@ -1296,7 +1290,7 @@
     "ButtonTrailer": "Trailer",
     "ButtonWebsite": "Webseite",
     "ChangingMetadataImageSettingsNewContent": "Änderungen an Metadaten- und Artwork-Einstellungen betreffen nur der Bibliothek neu hinzugefügte Inhalte. Um diese Änderungen auf existierende Medien anzuwenden müssen die Metadaten manuell aktualisiert werden.",
-    "CopyStreamURL": "Kopiere Streaminglink",
+    "CopyStreamURL": "Stream URL kopieren",
     "CopyStreamURLSuccess": "Link erfolgreich kopiert.",
```

Comment:
Is this an intentional change to the translation?

Reply:
Apparently someone added the key twice in this file, so my script replaced this instance with a value from the same key at the end of the file.